crates/serpen/tests/fixtures/ast_rewriting/regular_import_aliases/utils/config.py

```
@@ -0,0 +1,8 @@
+"""Configuration module for testing regular import aliases."""
+
+DEFAULT_CONFIG = {"debug": True, "timeout": 30}
+
+
+def get_config():
```

Ruff (0.11.9)

Missing return type annotation for public function `get_config`

(ANN201)